src/repository/resources/ipres.rs

```diff
@@ -1037,6 +1038,65 @@ impl AddressRange {
         }
     }
 
+    /// Convert a range into a set of V6 prefixes
+    pub fn to_v6_prefixes(self) -> impl Iterator<Item = Prefix> {
+        let mut start = self.min.to_bits();
+        let end = self.max.to_bits();
+        
+        std::iter::from_fn(move || {
+            if start > end {
+                return None;
+            }
+
+            let addr_host_bits = start.trailing_zeros();
```

Comment:
I think it would be nice to provide a short prose summary of what this is doing.